### ISSUE TITLE
I cant see any documentation of app.server in Electron website

### ISSUE THREAD

**Issue by omarojo** [theallmightyjohnmanning/electron-express#1]
I see you have a line for app.server in the main.js.

But I dont see any documentation for a `server` property in the Electron documentation website. Can you explain how that would work ?

**Comment by theallmightyjohnmanning** [comment]
Yes I haven't gotten around to writing a README.md for this yet, I'm sorry... don't npm install it yet because it's just an experimental project... still really useful though, clone the repo and all the node server code is in the app folder and then it's launched in an electron window... the app directory is basically just a static file server that serves up files on whatever port you configure in config.json .. I did that as just a trivial config seperation, and it's the only caveat that I've run into with this setup... when you build your electron app you have to put the config file in the app's root directory in order for it to run
